moveit_ros/perception/depth_image_octomap_updater/include/moveit/depth_image_octomap_updater/depth_image_octomap_updater.h

```diff
@@ -78,7 +78,7 @@ class DepthImageOctomapUpdater : public OccupancyMapUpdater
   image_transport::CameraPublisher pub_filtered_depth_image_;
   image_transport::CameraPublisher pub_filtered_label_image_;
 
-  rclcpp::Time last_update_time_;
+  rclcpp::Time last_update_time_ = rclcpp::Time(0, 0, RCL_ROS_TIME);
```

Comment:
```suggestion
  // Initialize to a large value so the first octomap update is not rejected due to a timeout
  rclcpp::Time last_update_time_ = rclcpp::Time(0, 0, RCL_ROS_TIME);
```

Reply:
If I remember correctly I did this to prevent getting an exception due to time source mismatch since the default one is `RCL_SYSTEM_TIME`

Reply:
OK. Basically, I'm asking you to leave a comment about why you made this change. Since it's not obvious from the code

Reply:
Done